testing/citest/spinnaker_testing/azure_scenario_support.py

```diff
@@ -49,17 +49,11 @@ def add_commandline_parameters(cls, scenario_class, builder, defaults):
         '--test_azure_vnet', 
         help='The name of the virtual network that contains the subnets')
     builder.add_argument(
-        '--test_azure_subnet1', 
-        help='The name of subnet 1')
+        '--test_azure_subnets', 
+        help='The names of subnets eparated by comma')
     builder.add_argument(
-        '--test_azure_subnet1_address', 
-        help='The address of subnet 1')
-    builder.add_argument(
-        '--test_azure_subnet2', 
-        help='The name of subnet 2')
-    builder.add_argument(
-        '--test_azure_subnet2_address', 
-        help='The address of subnet 2')
+        '--test_azure_subnet_address', 
+        help='The addresses of subnets eparated by comma')
```

Comment:
what is the relationship of the subnet and subnet address? Do these params correspond to one another? If so, maybe make this a single --test_azure_subnets parameter whose comma-separated values are in the form <name>=<address>.

Reply:
> what is the relationship of the subnet and subnet address? Do these params correspond to one another? If so, maybe make this a single --test_azure_subnets parameter whose comma-separated values are in the form =

The subnets are named by user while the subnets' addresses are related. In our case, they are 172.23.0.0/24 and 172.23.1.0/24. However there maybe more subnets in the virtual network whose addresses are 172.23.2.0/24...... So I think it may be better to leave it flexible.

Reply:
resolved.